behavior_metrics/brains/CARLA/brain_carla_segmentation_based_imitation_learning.py

```diff
@@ -0,0 +1,152 @@
+from torchvision import transforms
+from PIL import Image
+from brains.CARLA.utils.pilotnet_onehot import PilotNetOneHot
+from brains.CARLA.utils.test_utils import traffic_light_to_int, model_control
+from utils.constants import PRETRAINED_MODELS_DIR, ROOT_PATH
+from os import path
+
+import numpy as np
+
+import torch
+import torchvision
+import cv2
+import time
+import os
+import math
+import carla
+
+PRETRAINED_MODELS = ROOT_PATH + '/' + PRETRAINED_MODELS_DIR + 'il_models/'
```

Comment:
The `il_models` folder is missing. Should it be changed to `CARLA`?

Reply:
Yes, good catch!